src/evm/protocol/uniswap_v4/hooks/angstrom/hook_handler_creator.rs

```diff
@@ -0,0 +1,69 @@
+use std::collections::HashMap;
+
+use alloy::primitives::{aliases::U24, Address};
+
+use super::hook_handler::{AngstromFees, AngstromHookHandler};
+use crate::{
+    evm::protocol::uniswap_v4::hooks::{
+        hook_handler::HookHandler,
+        hook_handler_creator::{HookCreationParams, HookHandlerCreator},
+    },
+    protocol::errors::InvalidSnapshotError,
+};
+
+pub struct AngstromHookCreator;
+
+impl HookHandlerCreator for AngstromHookCreator {
+    fn instantiate_hook_handler(
+        &self,
+        params: HookCreationParams,
+    ) -> Result<Box<dyn HookHandler>, InvalidSnapshotError> {
+        let hook_address_bytes = params
+            .attributes
+            .get("hooks")
+            .ok_or_else(|| InvalidSnapshotError::MissingAttribute("hook_address".to_string()))?;
+
+        let pool_manager_address_bytes = params
+            .attributes
+            .get("balance_owner")
+            .ok_or_else(|| {
+                InvalidSnapshotError::MissingAttribute("pool_manager_address".to_string())
+            })?;
+
+        let angstrom_pool_config = params
+            .attributes
+            .get("angstrom_pool_config")
+            .ok_or_else(|| {
+                InvalidSnapshotError::MissingAttribute("angstrom_pool_config".to_string())
+            })?;
```

Comment:
aaff soo actually this doesn't exist.. the only extra attributes we will have will be the fees. This was done by Angstrom's integrator when we wanted to go for a more complex approach. For this simple one I think we only need the fees. See [here](https://github.com/propeller-heads/tycho-protocol-sdk/pull/313/files#diff-e5669645d0eaa11d92510f07aa2e5f355f87f0d85dbf7887d0e45a7ef40f0301R53-R70)